CONTRIBUTING.md

```diff
@@ -66,6 +66,7 @@ notification when you git push.
 We follow the conventions on [How to Write a Git Commit Message](http://chris.beams.io/posts/git-commit/).
 
 Be sure to include any related GitHub issue references in the commit message.  See
+<!-- wokeignore:rule=master -->
```

Comment:
This approach is inconsistent with the approach taken in the [apps plugin woke pr](https://github.com/vmware-tanzu/apps-cli-plugin/pull/394/files) - see the CONTRIBUTING.md file changes. 
Shouldn't this be appended to the line below rather than be above the line it's meant to "ignore"?

Reply:
`woke` treats both these styles in a similar way. It supports two flavors of in-line ignores:

1. ignore current-line and
2. ignore the immediate line following the comment.

I inadvertently chose 2nd option here. However, resulting action is still same.

Reply:
Thanks Ani - if both work, then LGTM.